src/Sylius/Behat/Client/Request.php

```diff
@@ -144,6 +144,28 @@ public static function custom(string $url, string $method): RequestInterface
         );
     }
 
+    public static function customUpdate(?string $section, ?string $resource, ?string $id = null, ?string $urlResource = null, ?string $token = null): RequestInterface
```

Comment:
`custom` is using ['CONTENT_TYPE' => 'application/merge-patch+json'] which does not work with this update.
And `update` is not designated for collection command but for item operation ( it needs id).

Other way to solve it, i can use  /** @var AbstractBrowser */ and create custom request $this->client->request(), but it might cause the all accountContext to be changed. Wdyt @GSadee ??

Reply:
Naming is misleading in this case. By default, POST operation suggests the `creation` of the object. This endpoint is declared as POST and it is a collection operation, which is not exactly true. We are operating on a particular item, but we wanted to skip user id from URI (and have `account` endpoint).